src/labkey/Exp.ts

```diff
@@ -38,7 +38,7 @@ export class ExpObject {
     /**
      * The person who created the ExpObject.
      */
-    createdBy: number;
+    createdBy: string;
```

Comment:
These are not LK user ids?

Reply:
They are user email addresses serialized in [ExperimentJSONConverter.serializeStandardProperties](https://github.com/LabKey/platform/blob/10719867f28056fc5e26a78467e33af992609927/api/src/org/labkey/api/exp/api/ExperimentJSONConverter.java#L154)

Reply:
Will users that aren't able to view emails not see `createdBy` information?